dockstore-integration-testing/src/test/java/io/dockstore/models/QuayModelTest.java

```diff
@@ -1,18 +1,66 @@
 package io.dockstore.models;
 
+import java.util.List;
+
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import io.swagger.quay.client.ApiClient;
+import io.swagger.quay.client.ApiException;
+import io.swagger.quay.client.Configuration;
+import io.swagger.quay.client.api.BuildApi;
+import io.swagger.quay.client.model.InlineResponse2001;
 import io.swagger.quay.client.model.PullRobot;
+import io.swagger.quay.client.model.QuayBuild;
 import org.junit.Assert;
 import org.junit.Test;
 
 import static io.dropwizard.testing.FixtureHelpers.fixture;
 
 public class QuayModelTest {
+
+    /**
+     * Tests that the swagger-generated Quay.io api can deserialize a sample PullRobot JSON (that isn't just null)
+     * @throws JsonProcessingException  If cannot deserialize
+     */
     @Test
     public void testPullRobot() throws JsonProcessingException {
         ObjectMapper objectMapper = new ObjectMapper();
         PullRobot pullRobot = objectMapper.readValue(fixture("fixtures/pullRobot.json"), PullRobot.class);
+        pullRobotAssertions(pullRobot);
+    }
+
+    /**
+     * Test that the swagger-generated Quay.io api can deserialize a sample InlineResponse2001 JSON that has pull_robot that isn't just null
+     * The sample JSON is the String response observed from testGetRepoBuildsWithPullRobot
+     * @throws JsonProcessingException If cannot deserialize
+     */
+    @Test
+    public void testInlineResponse2001() throws JsonProcessingException {
+        ObjectMapper objectMapper = new ObjectMapper();
+        InlineResponse2001 inlineResponse2001 = objectMapper.readValue(fixture("fixtures/inlineResponse2001.json"), InlineResponse2001.class);
```

Comment:
A little off-topic, but how do these inline response classes get named?
Is there a chance of breakage if we add new endpoints with new inline responses or is the 2001 deterministic in some way?

Reply:
I think swagger codegen names it based on the order of inline responses in the swagger.yaml.  I don't really see us adding new endpoints to the Quay API, but if we do, just add it to the bottom.  Even if we don't, it'll be obvious how to fix, compiler would complain and suggest what it should be.